noxfile.py

```diff
@@ -147,6 +128,25 @@ def run_pytest(
     )
 
 
+@nox.session(python=_SUPPORTED_PYTHONS)
+def get_unit_tests(session: nox.Session) -> None:
+    """Prints all unit test files that we want to run.
+
+    This is used by CircleCI to enable re-running only failed tests.
+    https://circleci.com/docs/rerun-failed-tests/
+    """
+    install_timed(
```

Comment:
i think this would very wasteful to run every time!

Reply:
because we use pytest to collect the files, we need to have pytest and all required dependencies when pytest collects files to be installed.

This does not add much more time because the dependencies will also be installed in the following step when the tests are actually executed.

Reply:
ok, not sure about this one... do you think it makes sense to not include it and change the order?

Reply:
> do you think it makes sense to not include it and change the order?

As in not include the `installed_time` in the actual test step section?